hnn_core/tests/test_compare_hnn.py

```diff
@@ -1,14 +1,23 @@
-import os.path as op
+from hnn_core import simulate_dipole, Network
 
-from numpy import loadtxt
-from numpy.testing import assert_array_equal
 
-from mne.utils import _fetch_file
-import hnn_core
-from hnn_core import simulate_dipole, Params, Network
+def run_simulation(params, dpl_master):
```

Comment:
missing docstring. When you run pytest, the docstrings are

Reply:
run_simulation is not called by pytest directly. I could name it _run_simulation if you'd suggest that instead.

Reply:
Or is the problem that pytest will automatically call it? It should get called from test_hnn_core() and if the script is executed directly.

Reply:
docstrings have been added in all test files now